README.md

```diff
@@ -39,6 +40,27 @@ the project, checkout the [open issues](https://github.com/operator-framework/ru
 [recent milestones](https://github.com/operator-framework/rukpak/milestones). There is a lot of development work ongoing
 and plenty of opportunities to get involved and [contribute](CONTRIBUTING.md)!
 
+## Install
+
+The recommended way of installing RukPak is via a tagged release from
+the [releases](https://github.com/operator-framework/rukpak/releases) page. There are detailed instructions provided in
+the release notes on how to install a particular release. The only requirement is to have a `kubectl` client available
```

Comment:
We can link directly to the `rukpak.yaml` file of the latest release like this:
```
kubectl apply -f https://github.com/operator-framework/rukpak/releases/latest/download/rukpak.yaml
```

Only caveat is that if we ever backport something and cut a release of an old patch version, I'm pretty sure GH will treat that as the latest. Basically "latest" to github is based on release timestamp, not semver.

Reply:
Ok that's fair -- we can include applying the rukpak.yaml file directly in the README -- I was unsure since there is a cert-manager dependency, but I can elaborate on that as well.

Reply:
I wonder if we should include a copy of the cert-manager manifests we depend on in our release. Then we could do:
```
kubectl apply -f https://github.com/operator-framework/rukpak/releases/latest/download/cert-manager.yaml
kubectl apply -f https://github.com/operator-framework/rukpak/releases/latest/download/rukpak.yaml
```

Reply:
Or we could just call cert-manager out as a dependency and not include any official instructions on installing it?

Reply:
Hmm, I'm not sure about vendoring cert-manager in, since it's a popular project chances are some users already have some version of it installed. We would benefit from just linking to an existing cert-manager on the cluster and avoid the issue of trying to install it if it's already present. We could attempt to conditionally install the dependency but it seems needlessly complicated. 

In our official documentation for the [v0.3 release](https://github.com/operator-framework/rukpak/releases/tag/v0.3.0) we do state a dependency on a specific version of cert-manager. I suppose we could remove that and include a note stating that the released version is dependent on cert-manager instead? 

I'm wondering if it makes sense to leave the release format as-is and from the root README just provide the rukpak installation link and explain that there is a dependency on cert-manager (with a link to how to install it on their doc site). That way users have flexibility in how they choose to install cert-manager.

Reply:
+1 to that approach. Except that for future releases, I think we could leave out the kubectl apply step that references the cert-manager manifest.

Reply:
Cool, I can reference that in #210 so we can consider it when tackling that issue.